snippets/assert-attribute-must-be-set.js

```diff
@@ -0,0 +1,48 @@
+/* Usage of this snippet:
```

Comment:
ユニペットを利用しているのは自明なのでちょっと冗長に感じました。
```suggestion
/* Usage:
```